src/main/kotlin/configuration/MultiplatformConfiguration.kt

```diff
@@ -16,9 +15,6 @@ internal fun Project.configureKotlinMultiplatformAndroid() {
             instrumentedTestVariant.sourceSetTree.set(KotlinSourceSetTree.test)
         }
     }
-    tasks.withType(KotlinCompile::class.java).all {
```

Comment:
`KotlinCompile` is actually a JVM compilation task (because Kotlin was originally JVM-specific), so this wasn't getting applied to other platforms' compilation tasks